crates/wasmtime/src/runtime/vm/interpreter.rs

```diff
@@ -319,6 +325,9 @@ impl InterpreterRef<'_> {
             (@get u8 $reg:ident) => (self.0[$reg].get_i32() as u8);
             (@get u32 $reg:ident) => (self.0[$reg].get_u32());
             (@get u64 $reg:ident) => (self.0[$reg].get_u64());
+            (@get f32 $reg:ident) => (self.0[$reg].get_f32());
+            (@get f64 $reg:ident) => (self.0[$reg].get_f64());
```

Comment:
In combination with dropping these methods below, could this be changed to `unreachable()` in the same manner as `i8x16`? Fixing this would otherwise require updates to the macro to have a separate "finger" for float registers as it has for integer registers right now, so this'll need more updates if floats are ever passed here.

Reply:
Done.